### ISSUE TITLE
Issue with certain spritefonts.

### ISSUE THREAD

**Issue by CloakedGames** [JujuAdams/scribble#114]
I have a few spritefonts which I am using with Scribble. One of my spritefonts does not display correctly. Some of the characters are repeated at the exclusion of others. See this screenshot, where the top one is incorrect, but the second one is fine.

![ScribblePixelfontIssue](https://user-images.githubusercontent.com/49505360/90571275-493efd00-e166-11ea-8289-fbcf29d8c0ec.PNG)

All the spritefonts are set up the same way. I checked that the collision mask was set up according to the documentation, and I tried putting the spritefont on its own texture page. There is one pixel of border around the font. Perhaps this is user error, but I am unable to solve this issue on my own at this point.

This particular issue is Scribble 6.0.10 for 2.3, which I updated to use today. The issue does not occur with Scribble 5.5.0 for 2.2 as far as I am aware, as it has been working for me previously.

Code:
```
//Create Event
scribble_init("", "fnt_pixelfont", false);
scribble_add_spritefont("fnt_pixelfont", "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz.,?!'()<>^*+=-/%:;_$", 1, 4);
scribble_add_spritefont("fnt_smallfont", "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz.,()'?!-/$", 1, 3);

//Draw Event
scribble_draw(x, y, "0123456789\nABCDEFGHIJKLMNOPQRSTUVWXYZ\nabcdefghijklmnopqrstuvwxyz\n.,?!'()<>^*+=-/%:;_$");
scribble_draw(x, y + 56, "[fnt_smallfont]0123456789\nABCDEFGHIJKLMNOPQRSTUVWXYZ\nabcdefghijklmnopqrstuvwxyz\n.,()'?!-/$[/f]");
```

Attached is a testing project which should demonstrate the issue. IDE v2.3.0.529
[ScribbleTesting.zip](https://github.com/JujuAdams/scribble/files/5093124/ScribbleTesting.zip)

**Comment by ahp275** [comment]
I came here to inquire about this exact issue. Glad it's not just me - I've been trying to figure out for a couple hours where I could've gone wrong! Scribble was working wonderfully when I was using GMS2.2, but today when I updated to 2.3 something seems to have gone awry. 

Here's my spritefont in the image editor:
![1spritefont](https://user-images.githubusercontent.com/47668326/90594688-a1025600-e1b0-11ea-817c-d309fc9092a7.jpg)

And here's the relevant code in my dialogue object (based on Juju's example project):
![2code](https://user-images.githubusercontent.com/47668326/90594804-e9ba0f00-e1b0-11ea-9dea-e9d973c69919.jpg)

But here's how it's showing up when I compile the game:
![3compiled](https://user-images.githubusercontent.com/47668326/90594827-01919300-e1b1-11ea-815b-e540578e85be.jpg)

I've tried deleting and reloading the sprite, rearranging the glyphs in the font string, etc. FWIW the built-in draw_text functions are working fine.
